pkg/backporter/httpcache_test.go

```diff
@@ -0,0 +1,109 @@
+package backporter
+
+import (
+	"bytes"
+	"errors"
+	"io/ioutil"
+	"net/http"
+	"testing"
+)
+
+//bugzillaCache tests
+
+func TestBugzillaCache(t *testing.T) {
+	testcases := []struct {
+		name      string
+		cachedKey string
+		value     []byte
+		absentKey string
+	}{
+		{
+			name:      "key 1",
+			cachedKey: "key1",
+			value:     []byte("Test string"),
+			absentKey: "keyx",
+		},
+	}
```

Comment:
There should be a testcase that checks that getting uncached responses works properly

Reply:
The test does that, since the first stage requests data which is uncached. 
The next stage checks whether the data is cached and if the cached data is returned the second time that data is requested